src/axes.js

```diff
@@ -70,24 +72,34 @@ export function autoAxisLabels(channels, scales, {x, y, fx, fy}, dimensions) {
 
 function autoAxisLabelsX(axis, scale, channels) {
   if (axis.labelAnchor === undefined) {
-    axis.labelAnchor = scale.type === "ordinal" ? "center"
-      : scale.reverse ? "left"
+    axis.labelAnchor = isOrdinalScale(scale) ? "center"
+      : isDescendingDomain(scale) ? "left"
       : "right";
   }
   if (axis.label === undefined) {
     axis.label = inferLabel(channels, scale, axis, "x");
   }
+  scale.label = axis.label;
```

Comment:
I think there’s a little internal confusion here on whether the label and labelAnchor properties live on the scale options, or on the axis options. Related to this, we should probably materialize the labelAnchor property in the exposed scale object, under the philosophy of materializing defaults? I’m going to think about this a bit more.